time start_timestamp
time end_timestamp
sensor_msgs/Imu[] measurements
